packages/live-update/ios/Plugin/LiveUpdate.swift

```diff
@@ -356,8 +356,10 @@ import CommonCrypto
             let temporaryZipFileUrl = self.cachesDirectoryUrl.appendingPathComponent(timestamp + ".zip")
             return (temporaryZipFileUrl, [.createIntermediateDirectories])
         }
-
-        AF.download(url, to: destination).validate().response { response in
+        var request = URLRequest(url: URL(string: url)!)
+        request.httpMethod = HTTPMethod.get.rawValue
+        request.timeoutInterval = Double(config.httpTimeout) / 1000.0
```

Comment:
Unfortunately, Capacitor does not offer a `getDouble` method: https://github.com/ionic-team/capacitor/issues/7634